test/user.test.js

```diff
@@ -214,6 +214,31 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('invalidates the user\'s accessToken when the user is deleted', function(done) {
+      User.create({ email: 'b@c.com', password: 'bar' }, function(err, user) {
```

Comment:
Unhandled `err`.

FWIW, I would personally rewrite this test to use promise API:

```js
it('...', function() {
  var userId;
  return User.create(...)
    .then(function(user) {
      userId = user.id;
      return User.login(...);
    })
    .then(function() {
      return User.deleteById(userrId);
    })
    // ...
});
```

If you prefer to stick to callbacks, then consider using `async.series` to avoid xmas tree in your code.

```js
it('...', function(done) {
  asyc.series([
    function(next) { User.create(...); },
    function(next) { User.login(...); },
    //...
  ], done);
});
```